src/layouts/AppLayout/index.tsx

```diff
@@ -25,12 +26,21 @@ const MainStyle = styled("div")(({ theme }) => ({
 
 export default function DashboardLayout() {
   const [open, setOpen] = React.useState(false);
+
+  const [{ data }] = useMeQuery({
```

Comment:
One of the design decisions that we are trying to enforce is fetching and subscribing to data on a container level, and then containers would feed the data to dumb and reusable components. In this case, we would use useMeQuery on individual containers i.e NavigationBarContainer and SidebarContainer and only subscribe to the data which is needed by those containers. There are some benefits to this design style and perhaps some draw backs.. let me know what you think of this. We can discuss this in our next call as well.

Reply:
Agreed. I noticed the the data being cached when using the useMeQuery, so since the data is not fetched everytime using useMeQuery does make sense.